include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -131,12 +131,8 @@ struct buffered_piece_collection
 
     // Robust ring/polygon type, always clockwise
     typedef geometry::model::ring<robust_point_type> robust_ring_type;
-    typedef geometry::model::box<robust_point_type> robust_box_type;
 
-    typedef typename default_comparable_distance_result
-        <
-            robust_point_type
-        >::type robust_comparable_radius_type;
+    typedef geometry::model::box<point_type> normal_box_type;
```

Comment:
Maybe simply `box_type`? The robust ones will go anyway.

Reply:
I renamed it in the next PR (which is nearly ready) because indeed the robust one is gone then.